source/synthDrivers/sapi5.py

```diff
@@ -35,6 +36,14 @@
 from ._sonic import SonicStream, initialize as sonicInitialize
 
 
+class SPAudioState(IntEnum):
```

Comment:
I think we should avoid making this public so we don't have to deprecate / remove it again when we remove WinMM support

```suggestion
class _SPAudioState(IntEnum):
```